tests/performance/gatling_tests/build.gradle

```diff
@@ -26,6 +26,13 @@ repositories {
     mavenCentral()
 }
 
+sourceSets {
+    gatling {
+        scala.srcDirs = ["src/gatling/scala/"]
+        resources.srcDirs = ["src/gatling/resources/"]
```

Comment:
What would the default setup look like? Should we move our stuff to conform to the default setup?

Reply:
We could move, yes, figured I'd land the update and make sure I didn't blow up the world then I can do another one to move us to their new layout. I will open a issue for that now.